### ISSUE TITLE
vNext - Magnifying glass works only once, then you can't use it again, and can't switch tool.

### ISSUE THREAD

**Issue by JamesAPetts** [cornerstonejs/cornerstoneTools#634]
As per the title.
You can switch to another tool fine as long as you activated the magnifying glass but didn't use it.

**Comment by lordfuoco** [comment]
Hi! I would like to pick this up :)

**Comment by dannyrb** [comment]
👋 Thanks for the PR, @lordfuoco! I've merged your fix. @JamesAPetts if you have any follow-up issues or comments, please feel free to create a new issue or add to this one. Closing for now ^_^
